packages/manager/apps/carrier-sip/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@ovh-ux/manager-carrier-sip-app",
   "version": "1.0.4",
   "private": true,
+  "description": "Carrier SIP Trunk standalone application.",
```

Comment:
I don't think we need `.`, do we ?

Reply:
It's not mandatory but It could be helpful I would say.

Description can be enhanced.